sdk/monitor/monitor-query/review/monitor-query.api.md

```diff
@@ -81,6 +59,12 @@ export interface LogsColumn {
 // @public
 export type LogsColumnType = string;
 
+// @public
+export interface LogsErrorInfo extends Error {
+    code: string;
+    message: string;
```

Comment:
should this have a  status which is always `LogsQueryResultStatus.FAILURE`?

How would users determine if this is a failure or a success?